pymatviz/colormaps.py

```diff
@@ -0,0 +1,122 @@
+"""Generate custom colormaps."""
+
+
+from __future__ import annotations
+
+import matplotlib
+import numpy as np
+from matplotlib.colors import Colormap, LinearSegmentedColormap, ListedColormap
+
+
+def combine_two(
+    cmaps: list[Colormap | str],
+    node: float = 0.5,
+    N: int = 256,
+    reverse: bool = False,
+) -> Colormap:
```

Comment:
isn't this very similar to `matplotlib.colors.LinearSegmentedColormap`? do we really need a custom helper function?

Reply:
Yes it is quite similar, only one main difference here: to assign "[nodes](https://matplotlib.org/stable/users/explain/colors/colormap-manipulation.html)" so that the boundary between children colormaps could be adjusted (for example a composite "two in one" colormap might not necessarily have each child taking up 50% of the colormap, but could be adjusted to say 20%/80%). Let me know if there is more straightforward way to achieve this.

Reply:
I'm trying to find an existing function for this and may remove this part of code after that😊.

Reply:
I might go ahead and PR this function to some low level python package (`cmasher` for example), and remove `combine_two` from here after that, to keep `pymatviz` central to high level ops.